python-sdk/pachyderm_sdk/client.py

```diff
@@ -222,7 +223,9 @@ def from_pachd_address(
         )
 
     @classmethod
-    def from_config(cls, config_file: Union[Path, str] = CONFIG_PATH_LOCAL) -> "Client":
+    def from_config(cls,
+        config_file: Union[Path, str] = os.environ.get(PACH_CONFIG_ENV, CONFIG_PATH_LOCAL)
```

Comment:
There's nothing wrong with doing it this way, I just think it would be clearer if we instead added
```python
DEFAULT_CONFIG = os.environ.get(PACH_CONFIG_ENV, CONFIG_PATH_LOCAL)
```
to `constants.py` and used that value as the default here. That way the user can at least check what the default value is set to here. 

Technically, allbeit there's a slim chance of this happening, the value of the `PACH_CONFIG_ENV` environment variable could change in the time between this class being instantiated and the user executing `os.environ.get(PACH_CONFIG_ENV, CONFIG_PATH_LOCAL)` to check what the default value is here.

Reply:
Done!